config/landing_pages.yml

```diff
@@ -154,7 +154,7 @@ shared:
   ### Working patterns
   flexible-working-jobs-in-schools:
     working_patterns:
-      - flexible
```

Comment:
This ensures that if users follow old links to the landing pages for flexible, job share or term time jobs it filters by part time jobs